spec/lcm/integration/shared_contexts_for_lcm.rb

```diff
@@ -127,40 +129,40 @@ def create_workspace_csv(workspaces, client_id_column)
     label.save
 
     prod_connection_parameters = {
-        username: @config[:username],
-        password: @config[:password],
-        server: "https://#{@config[:prod_server]}"
+      username: @config[:username],
+      password: @config[:password],
+      server: "https://#{@config[:prod_server]}"
     }
     @prod_rest_client = GoodData.connect(prod_connection_parameters)
     @prod_ads = ConfigurationHelper.create_development_datawarehouse(client: @prod_rest_client,
```

Comment:
Similar code found in shared_contexts_for_lcm.rb:89, shared_contexts_for_lcm.rb:131 (mass = 32)